editors/emacs/lambdapi-proofs.el

```diff
@@ -23,12 +40,34 @@
                                     (type (plist-get hyp :htype)))
                                 (format "%s: %s\n" name type)))
                             (reverse hs)))
+                 (goalnum  0)
                  (goalsstr (mapcar
                             (lambda (goal)
-                              (let ((id (plist-get goal :gid))
-                                    (type (plist-get goal :type)))
-                                (format "%s\nGoal %d: %s\n\n"
-                                        lp-goal-line-prefix id type)))
+                              (let* ((id (plist-get goal :gid))
```

Comment:
If I understand well, you make each goal clickable in this mapcar. Could you please just add a comment that says so?